src/runtime/classobj.cpp

```diff
@@ -1516,6 +1525,7 @@ void setupClassobj() {
     instance_cls->giveAttr("__int__", new BoxedFunction(boxRTFunction((void*)instanceInt, UNKNOWN, 1)));
     instance_cls->giveAttr("__long__", new BoxedFunction(boxRTFunction((void*)instanceLong, UNKNOWN, 1)));
     instance_cls->giveAttr("__float__", new BoxedFunction(boxRTFunction((void*)instanceFloat, UNKNOWN, 1)));
+    instance_cls->giveAttr("__complex__", new BoxedFunction(boxRTFunction((void*)instanceComplex, UNKNOWN, 1)));
```

Comment:
Well, most oldstyle-class attribute lookups get automatically proxied.  It's just for the attributes that have capi slots where we need to provide a wrapper implementation, and I don't think there's one for `__complex__`.  Could you try the test without this and see if it works?